nerfstudio/scripts/exporter.py

```diff
@@ -485,6 +485,8 @@ class ExportGaussianSplat(Exporter):
     """Rotation of the oriented bounding box. Expressed as RPY Euler angles in radians"""
     obb_scale: Optional[Tuple[float, float, float]] = None
     """Scale of the oriented bounding box along each axis."""
+    ply_color_mode: Literal["sh_coeffs", "rbg"] = "sh_coeffs"
+    """If True, outputs for rendering in renderers that support SH0 rendering."""
```

Comment:
Typo (`rbg` instead of `rgb`). Also, probably good to update the description.

```suggestion
    ply_color_mode: Literal["sh_coeffs", "rgb"] = "sh_coeffs"
    """If "rgb", export colors as red/green/blue fields. Otherwise, export colors as spherical harmonics coefficients."""
```